.github/workflows/renew-env.yaml

```
@@ -0,0 +1,75 @@
+name: PR-label -> renew dev env
+
+on:
+  pull_request:
+    types: [labeled]
+
+concurrency:
+  group: renew-env-${{ github.ref }}
+
+jobs:
+  renew:
+    name: Renew dev environment
+    runs-on: ubuntu-latest
+    continue-on-error: true # Ignore errors
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    steps:
+      - name: Check PR env labels
+        uses: voiceflow/pr-label-match@master
+        with:
+          regex: env-
+        id: envNameLabel
+
+      - name: Check for renew env request
+        uses: Dreamcodeio/pr-has-label-action@master
+        id: renewEnvLabel
+        with:
+          label: renew-env
+
+      - name: Remove renew-env label (if exist)
+        uses: buildsville/add-remove-label@v1
+        if: steps.renewEnvLabel.outputs.hasLabel == 'true' && steps.renewEnvLabel.outputs.label == ''
+        with:
+          token: ${{secrets.GH_SA_TOKEN}}
+          label: renew-env
+          type: remove
+
+      - name: Install vfcli
+        if: (github.event.action == 'labeled' && steps.renewEnvLabel.outputs.hasLabel == 'true' && steps.envNameLabel.outputs.label != '')
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_SA_TOKEN }}
+        run: |
+          API_URL="https://$GITHUB_TOKEN:@api.github.com/repos/voiceflow/vfcli"
+          ASSET_ID=$(curl $API_URL/releases/latest | jq -r '.assets[3].id')
+          curl -J -L -H "Accept: application/octet-stream" "$API_URL/releases/assets/$ASSET_ID" --output vfcli.tar.gz
+          tar -xf vfcli.tar.gz
+
+      - name: Renew the dev env if PR closed not creating and the label exists
+        if: (github.event.action == 'labeled' && steps.renewEnvLabel.outputs.hasLabel == 'true' && steps.envNameLabel.outputs.label != '')
```

The check is redundant because the workflow is only triggered by that event
```suggestion
        if: (steps.renewEnvLabel.outputs.hasLabel == 'true' && steps.envNameLabel.outputs.label != '')
```

---

fixed